airbyte-integrations/connectors/source-braintree/pyproject.toml

```diff
@@ -21,7 +21,7 @@ airbyte-cdk = "^0.50"
 braintree = "^4.28.0"
 
 [tool.poetry.scripts]
-source-exchange-rates = "source_braintree.run:run"
```

Comment:
🤦 

oof - I copied this file from that source, missed the rename here

Reply:
Bruh ☠️